package.json

```diff
@@ -15,8 +15,8 @@
     "build": "./build.sh",
     "build:feeds": "node scripts/generate-blog-feeds.js",
     "lint": "npm run lint:server",
-    "lint:server": "DEBUG=eslint:cli-engine npx eslint --max-warnings=0 --ext .js,.jsx .",
-    "lint:static-site": "cd static-site && DEBUG=eslint:cli-engine npx eslint --max-warnings=0 --ext .js,.jsx,.ts,.tsx .",
+    "lint:server": "DEBUG=eslint:cli-engine npx eslint --max-warnings=0 .",
+    "lint:static-site": "cd static-site && DEBUG=eslint:cli-engine npx eslint --max-warnings=0 .",
```

Comment:
Tangential, but can `--max-warnings=0` move into the eslint configs?

Reply:
[Not possible](https://github.com/eslint/eslint/issues/16804#issuecomment-1460887876) 😕